content/blog/2016/2016-10-27-xunit-reporting.adoc

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: "xUnit and Pipeline"
+tags:
+- pipeline
+- plugins
+- xunit
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:http://cloudbees.com[CloudBees].
+
+The
+link:https://wiki.jenkins-ci.org/display/JENKINS/JUnit+Plugin[JUnit plugin]
+is the go-to test result reporter for many Jenkins projects,
+but the it is not the only one available.  The
+link:https://wiki.jenkins-ci.org/display/JENKINS/xUnit+Plugin[xUnit plugin]
+is a viable alternative that supports JUnit and many other test result file formats.
+
+=== Introduction
+
+No matter the project, you need to gather and report test results.
```

Comment:
Basically, as long as I'm going to talk about test reporting in Jenkins, give a mini tour.

Trendgraphs: like the ones in the saucelabs blog post? 
https://jenkins.io/images/post-images/2016-08-29/tests-report.png

Test failures: 
You mean the "testReport" page, right? 

<img width="929" alt="screen shot 2016-09-12 at 4 20 53 pm" src="https://cloud.githubusercontent.com/assets/1958953/19788458/284f7b76-9c5d-11e6-8ca7-10cb8c90c4ab.png">